fahrplan/API.py

```diff
@@ -59,10 +59,10 @@ def _parse_section(con_section, connection):
     section = {}
     section['station_from'] = departure['station']['name']
     section['station_to'] = arrival['station']['name']
-    section['travelwith'] = journey["name"]
+    section['travelwith'] = journey["name"] if journey!=None else ""
```

Comment:
As `None` is a singleton in Python, the comparison must be `if journey is not None`.